tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -65,7 +73,7 @@
         <flag attr="RAmem" val0="3" val1="1"/>
     </global>
     <fasta2cmap>
-        <flag attr="enzyme" val0="BspQI" display="Enzyme" group="FASTA to CMAP digestion" description="Define single enzyme for in-silico FASTA to CMAP digestion. Avalible enzymes: BspQI, BbvCI, BsmI, BsrDI, BssSI, DLE1."/>
+        <flag attr="enzyme" val0="CTTAAG" display="Enzyme" group="FASTA to CMAP digestion" description="Define single enzyme for in-silico FASTA to CMAP digestion. Avalible enzymes: BspQI, BbvCI, BsmI, BsrDI, BssSI, DLE1."/>
```

Comment:
Why is this changed?

Reply:
Because the VGP configuration uses that sequence (corresponding to a specific restriction enzyme); the input files used in the test uses a different one.

Reply:
ok, maybe that parameter should be made configureable?

Reply:
I think that It wouldn't be necessary (except for testing purposes). But if you consider so I'll include it.